### ISSUE TITLE
"Selfie-stick" video object

### ISSUE THREAD

**Issue by nampereira** [conix-center/ARENA-core#311]
**Is your feature request related to a problem? Please describe.**
Enable showing video streams from a separate client. This would enable, for example, a video of a presenter, or of a user from outside his device.

**Describe the solution you'd like**
An object that shows an arbitrary jitsi video stream. 

**Describe alternatives you've considered**
Let the local video cube be swapped out for a jitsi stream could be useful too.

**Comment by nampereira** [comment]
It seems that it would be useful to define the user presence video appearance to either be the current video cube or a simple plane.

**Comment by nampereira** [comment]
Here is an example that displays the video of jitsi user display name 'nuno' on a plane:

```
{
  "object_id": "avideoexample",
  "persist": true,
  "type": "object",
  "action": "update",
  "data": {
    "object_type": "plane",
    "height": 1,
    "width": 1,
    "material": {
      "color": "#7f7f7f",
      "side": "double"
    },
    "position": {
      "x": 0,
      "y": 2,
      "z": -5
    },
    "rotation": {
      "x": 0,
      "y": 0,
      "z": 0
    },
    "jitsi-video": {
      "displayName": "nuno"
    }
  }
}
```

![image](https://user-images.githubusercontent.com/3504501/125852642-49bb2a24-3900-4d8c-9f8c-3a8ef9cffdf9.png)

![image](https://user-images.githubusercontent.com/3504501/125852799-df873cb5-b822-4127-9741-428cd56c813e.png)

**Comment by nampereira** [comment]
Added by #312, #318, #319

**Comment by DaAwesomeP** [comment]
The object should probably go back to its original material if the Jitsi client disconnects. Currently it just freezes on the last frame. This way new users loading the scene and existing ones will always see the same thing.

**Comment by nampereira** [comment]
Quick fix in #6820910 removes video source when user leaves.

**Comment by nampereira** [comment]
> The object should probably go back to its original material if the Jitsi client disconnects. Currently it just freezes on the last frame. This way new users loading the scene and existing ones will always see the same thing.

Quick fix that removes video source when user leaves:
https://github.com/conix-center/ARENA-core/commit/6820910b9fe3ba183772bfdfc09ed438b9dd9b37
